src/main/resources/db/changelog/changes/07-insert-admin-in-users-table.yaml

```diff
@@ -1,29 +1,29 @@
-databaseChangeLog:
-  - changeSet:
-      id: test-insert-into-users-table
-      author: aleksandrparkhomovsky@gmail.com
-      changes:
-        - insert:
-            tableName: users
-            columns:
-              - column:
-                  name: email
-                  value: invader@example.com
-              - column:
-                  name: first_name
-                  value: Starlord
-              - column:
-                  name: last_name
-                  value: PonyDestroyer
-              - column:
-                  name: password
-                  value: I$J(QJ#T(QIG#Q*G(Q#JIRARJAK@(JRHI==wf
-        - insert:
-            tableName: users_roles
-            columns:
-              - column:
-                  name: user_id
-                  valueNumeric: 2
-              - column:
-                  name: role_id
-                  valueNumeric: 2
+databaseChangeLog:
+  - changeSet:
+      id: test-insert-into-users-table
+      author: aleksandrparkhomovsky@gmail.com
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: invader@example.com
+              - column:
+                  name: first_name
+                  value: Starlord
+              - column:
+                  name: last_name
+                  value: PonyDestroyer
+              - column:
+                  name: password
+                  value: I$J(QJ#T(QIG#Q*G(Q#JIRARJAK@(JRHI==wf
+        - insert:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  valueComputed: 1
```

Comment:
this is bad idea to change sth in migration scripts that were pushed. If you have a db on prod, this change will brake your service at all.